docs/source/install.rst

```diff
@@ -0,0 +1,129 @@
+Installation
+============
+
+Clone the repository:
+
+.. code-block:: bash
+
+    $ git clone git@github.com:bigchaindb/bigchaindb-examples.git
+
+Go into it!
+
+.. code-block:: bash
+
+    $ cd bigchaindb-examples
+
+We now document two options:
+
+* One that is considered easy, if you consider using docker to be easy.
+* One that is considered to be hard, if you consider installing different
+  components, and required dependencies to be rarely a smooth process.
+    
+
+The Easy Way
+------------
+Just make sure you have recent versions of `docker engine`_ and
+`docker-compose`_, e.g.:
+
+.. code-block:: bash
+    
+    $ docker --version
+    Docker version 1.11.1, build 5604cbe
+
+    $ docker-compose --version
+    docker-compose version 1.7.0, build 0d7bf73
+
+
+Build the images:
+
+.. code-block:: bash
+
+    $ docker-compose build
+
+Start ``RethinkDB`` in the backgroud:
+
+.. code-block:: bash
+
+    $ docker-compose up -d rdb
+
+Configure ``BigchaninDB``:
+
+.. code-block:: bash
+
+    $ docker-compose run --rm bdb bigchaindb -y configure
```

Comment:
Did this work for you? My docker keeps complaining that `bigchaindb` is a directory (since I guess it's been mounted as one?) so I had to add `bigchaindb -yc /root/.bigchaindb/.bigchaindb_examples configure`.